bin/brew

```diff
@@ -1,7 +1,10 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
All of this stuff is built-in into `bash` (try `type [`), so the performance shouldn't be an issue. As I stated, since this is now bash code, using `[[` feels more idiomatic. No other gain than that. But for me that would be enough (especially for the newly written code).